types/report-renderer.d.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH.Renderer {
+    export function renderFullReport(lhr: LH.Result, options?: ReportRendererOptions): HTMLElement;
```

Comment:
```suggestion
    export function render(lhr: LH.Result, options?: ReportRendererOptions): HTMLElement;
```

imo that this would render the full report can be assumed.

Reply:
Given our imminent introduction of the difference between a single LHR report and a flow report of multiple LHRs, I like being explicit here, but the name likely changes in that case anyway.

compromise: `renderReport` for now? (parallels `renderReport<whatever noun we use>`)
`renderNavigationReport` eventually?